src/LinearOperators.jl

```diff
@@ -16,11 +16,15 @@ type LinearOperatorException <: Exception
 end
 
 # when indexing, Colon() is treated separately
-typealias LinearOperatorIndexType Union{UnitRange{Int}, StepRange{Int, Int}, AbstractVector{Int}}
+const LinearOperatorIndexType = Union{UnitRange{Int}, StepRange{Int, Int}, AbstractVector{Int}}
```

Comment:
Apparently :). Didn't expect it to be, but it didn't raise any errors. Unless it's untested